Kitodo/src/main/java/org/kitodo/production/services/dataformat/MetsService.java

```diff
@@ -46,11 +54,76 @@ private MetsService() {
                 MetsXmlElementAccessInterface.class).loadModule();
     }
 
-    public Workpiece load(InputStream in) throws IOException {
-        return metsXmlElementAccess.read(in);
+    /**
+     * Function for loading METS files from URI.
+     *
+     * @param uri
+     *            address of the file to be loaded
+     * @return loaded file
+     * @throws IOException
+     *             if reading is not working (disk broken, ...)
+     */
+    public Workpiece load(URI uri) throws IOException {
```

Comment:
I would a MetsService.load function expect to load METS (which is somehow happening) but its returning a Workpiece.
Should the Service be called WorkpieceService? Should the Workpiece Class be called "METS"? or at least call the Method "loadWorkpiece"